renderer/reducers/channels.js

```diff
@@ -843,7 +844,12 @@ const channelSortSelector = state => state.channels.sort
 const channelSortOrderSelector = state => state.channels.sortOrder
 const filterSelector = state => state.channels.filter
 const nodesSelector = state => networkSelectors.nodes(state)
-const viewModeSelector = state => state.channels.viewMode
+const viewModeSelector = createSelector(
+  settingsSelectors.currentConfig,
+  currentConfig => {
+    return get(currentConfig, 'channels.viewMode', config.channels.viewMode)
```

Comment:
`currentConfig` contains the default config merged with local config overrides, and with database overrides applied. So, you shouldn't need to set the default here as it's redundant.

Reply:
OK, will update.

Reply:
Alright, done.